### PR TITLE
Using URLs with start-build --from-file/--from-dir

### PR DESCRIPTION
**Pull request by mmilata**
Documentation for openshift/origin#11811, please don't merge before that PR is merged.

### CODE REVIEW

File: cli_reference/basic_cli_operations.adoc [hunk 1]
```diff
@@ -333,6 +333,15 @@ one in the build source. For example, *_pom.xml_* or *_Dockerfile_*.
 ----
 $ oc start-build --from-file=<file_name>
 ----
+Download the binary input using http/https instead of reading it from the file
+system:
```
Comment (ahardin-rh) [inline cli_reference/basic_cli_operations.adoc]: Include an empty line above the code block per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks

File: cli_reference/basic_cli_operations.adoc [hunk 2]
```diff
@@ -333,6 +333,15 @@ one in the build source. For example, *_pom.xml_* or *_Dockerfile_*.
...
+----
+$ oc start-build --from-file=<file_url>
+----
+Download an archive and use its contents as the build source:
+----
```
Comment (ahardin-rh) [inline cli_reference/basic_cli_operations.adoc]: Same here

File: dev_guide/builds.adoc [hunk 3]
```diff
@@ -1168,6 +1168,15 @@ effectively be ignored and replaced by what the client sends.
 disabled, since `*Binary*` and `*Git*` are mutually exclusive, and the data in
 the binary stream provided to the builder takes precedence.
 
+Instead of a file name you can pass URL with http or https schema to
```
Comment (ahardin-rh) [inline dev_guide/builds.adoc]: comma after name

File: dev_guide/builds.adoc [hunk 4]
```diff
@@ -1168,6 +1168,15 @@ effectively be ignored and replaced by what the client sends.
...
+`--from-file` and `--from-dir`.  When using  `--from-file` with a URL, the name
```
Comment (ahardin-rh) [inline dev_guide/builds.adoc]: Only one space between sentences.

**Comment by mmilata** [comment]
Fixed found issues & updated to match the Origin PR. PTAL @ahardin-rh

**Comment by ahardin-rh** [comment]
LGTM! I am adding a to_followup label, as I will make a few minor follow-up style edits in a separate PR. Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../cli_reference/basic_cli_operations.adoc#cli-reference-basic-cli-operations[Developer CLI Operations]
|Added information about `oc start-build --from-file` and `oc start build --from-dir`.
%
|xref:../dev_guide/builds.adoc#dev-guide-builds[Builds]
|Added information about using URLS with `oc start-build --from-file` and `oc start build --from-dir`.
%

**Pull request merged**